Python/Product/EnvironmentsList/PipExtension.xaml.cs

```diff
@@ -252,6 +263,10 @@ PipExtensionProvider provider
             _provider.InstalledPackagesChanged += PipExtensionProvider_InstalledPackagesChanged;
 
             IsPipInstalled = _provider.IsPipInstalled ?? true;
+            ShowSecurityWarning =
+                provider._packageManager.UniqueKey == "pip" &&
```

Comment:
I think we can make this not pip specific.

Reply:
I didn't want to to show for conda package management, because there it's not dependent on the python version of the environment, but rather on the version of the global conda.exe.

Using latest Anaconda, if I create a python=2.6 env, I can successfully install conda packages into it.